runtime-light/utils/panic.cpp

```diff
@@ -13,11 +13,10 @@
 void critical_error_handler() {
   constexpr const char *message = "script panic";
   const auto &platform_ctx = *get_platform_context();
-  auto &component_ctx = *get_component_context();
```

Comment:
Why'd you delete it?

Reply:
Because it can be called on image creation step